### ISSUE TITLE
Origins 1.17.1 does not work with optifabric

### ISSUE THREAD

**Issue by ewRetro** [apace100/origins-fabric#414]
I'm not sure if it's on purpose so some features of the mod can work but you can't use Optifabric with Origins.

**Comment by ChrysanthCow** [comment]
Excerpt from the !optifine command in origins discord

```
OptiFine is closed-source and handles Minecraft code very invasively. This means it breaks many mods and there's not much mod developers can do about it. From the developer of OptiFabric, the compatibility mod that makes using OptiFine possible on Fabric:

Please note the in the 1.16 (and somewhat 1.15) versions there are a lot of incompatiblities with a great number of mods, this is due to the way optifine changes the vanilla code in increasingly invasive ways. Fixing these crashes isnt easy or fun and takes a lot of time that I dont really have.

Therefore OptiFine won't work with Origins. If you are looking for alternatives, Fabric has a couple of options which you can try, depending on what you use OptiFine for. See here for a comprehensive list:
https://gist.github.com/LambdAurora/1f6a4a99af374ce500f250c6b42e8754
```

Tl;dr optifine is made in a way where Apace can't really account to make it compatible, the best solution is to use optifine alternatives. The list being here:
https://gist.github.com/LambdAurora/1f6a4a99af374ce500f250c6b42e8754
